readthedocs/oauth/notifications.py

```diff
@@ -96,5 +99,21 @@
         ),
         type=ERROR,
     ),
+    Message(
+        id=MESSAGE_OAUTH_DEPLOY_KEY_ATTACHED_FAILED_ON_MANUAL_IMPORTED_PROJECT,
+        header=_("Failed to add deploy key to project"),
+        body=_(
+            textwrap.dedent(
+                """
+                We were unable to find this repository in your {{ provider_name }} account.
+                If this is a private repository, you'll need to
+                <a href="https://docs.readthedocs.io/page/guides/importing-private-repositories.html">
+                  add the deploy key manually
+                </a>.
+                """
```

Comment:
I think we want to communicate this in a more direct way mentioning manual import or similar.

> When adding a project manually, you need to add the deploy key manually. Follow this guide to add the deploy key manually.

ccing @agjohnson since I think he will have a better copy for this notification.

Reply:
That is closer to what I was thinking, however we should also cover public repositories with this notification. All manually created projects will still need to set up a webhook manually, and private repos will need a deploy key. The notification should probably more generically mention set up steps, the above is specific to deploy keys.

> When adding a project manually, your repository will require extra configuration steps. Follow this guide to finish set up of your project.

Reply:
Don't we already mention the extra steps in the new dashboard?
 
![Screenshot 2024-10-01 at 17-03-09 Add project - Read the Docs](https://github.com/user-attachments/assets/e6ac352f-b61a-41af-b4af-af26e0df960e)

That links to https://docs.readthedocs.io/en/stable/intro/add-project.html, which doesn't mention anything about SSH keys for private repos, we should mention that.

Anyway, I think it would be useful to show a notification after the project is imported, as a reminder to the user. But don't think we need to do that when adding an ssh key.

Reply:
We mention that extra steps are required beginning project creation, but still want _some_ notification afterwards that these are problems the user must address.

I would not mention the GitHub account, as this is not required to even be connected, and I wouldn't mention deploy keys specifically, as the project/repo is missing webhooks at this point too. The notification should be inclusive to both, as both.

Eventually, this could be onboarding steps with individual actions required, but we're not quite there yet.

Reply:
It sounds like manually created projects will still end up in a state where they need a webhook attached manually, but we aren't giving the user this feedback after creating their project. Correct?

Reply:
Yeah, we need that notification, but it shouldn't be triggered from the SSH set up code, since manual imported projects also require setting up a webhook.